docs/src/main/asciidoc/rabbit/rabbit_overview.adoc

```diff
@@ -493,12 +493,13 @@ public Consumer<Thing> input() {
 spring.cloud.stream.bindings.input-in-0.destination=super
 spring.cloud.stream.bindings.input-in-0.group=test
 spring.cloud.stream.bindings.input-in-0.consumer.instance-count=3
+spring.cloud.stream.bindings.input-in-0.consumer.concurrency=3
 spring.cloud.stream.rabbit.bindings.input-in-0.consumer.container-type=STREAM
 spring.cloud.stream.rabbit.bindings.input-in-0.consumer.super-stream=true
 ----
 ====
 
-The framework will create a super stream named `super`, with 3 partitions.
+The framework will create a super stream named `super`, with 9 partitions.
```

Comment:
Are those partition going to be distributed between instances according to their `index`?
Or how does it work, please?

Reply:
It works similar to Kafka; when there is one instance, each of 3 consumers will get 3 partitions; when the instances increase, the partitions will be redistributed; when there are 3 instances, there will be 9 consumers each consuming from 1 queue (partition) and the other 8 consumers in wait mode. This is how it looks on one queue with one instance running...

<img width="996" alt="Screen Shot 2022-10-05 at 2 20 14 PM" src="https://user-images.githubusercontent.com/483832/194133634-6184f4bc-40db-4730-b01c-f97af5c8f031.png">